terraform/modules/jenkins/security.tf

```diff
@@ -57,41 +33,22 @@ resource "aws_security_group" "ec2_internal" {
   }
 }
 
-resource "aws_security_group" "ec2_cloudfront_global" {
-  name        = "${var.app_name}-ec2-security-group-global"
-  description = "Allows access to Jenkins from global cloudfront IPs"
+resource "aws_security_group" "jenkins_alb_group" {
+  name        = "${var.app_name}-alb-security-group"
+  description = "Controls access to the Jenkins load balancer"
   vpc_id      = aws_vpc.main.id
-
   ingress {
     protocol    = "tcp"
-    from_port   = 80
-    to_port     = 80
-    cidr_blocks = data.aws_ip_ranges.cloudfront_ranges_global.cidr_blocks
+    from_port   = 443
+    to_port     = 443
+    cidr_blocks = ["89.197.114.197/32"]
```

Comment:
Where does this range come from?

Reply:
It's the external IP from my machine. I checked with IT and there's also 89.197.114.197/32 so I've added this one as well. They're unlikely to change and they're not configured by us so I think this is ok.

Reply:
I see you've moved this to SSM. :+1:

Reply:
I've put them in the parameter store although if we end up with a private repo with the config json, I'll switch it over to use this.